contracts/v2/FiatTokenV2.sol

```diff
@@ -28,31 +28,26 @@ import { FiatTokenV1_1 } from "../v1.1/FiatTokenV1_1.sol";
 import { AbstractFiatTokenV2 } from "./AbstractFiatTokenV2.sol";
 import { EIP712 } from "../util/EIP712.sol";
 import { EIP712Domain } from "./EIP712Domain.sol";
-import { GasAbstraction } from "./GasAbstraction.sol";
-import { Permit } from "./Permit.sol";
+import { EIP3009 } from "./EIP3009.sol";
+import { EIP2612 } from "./EIP2612.sol";
 
 /**
  * @title FiatToken V2
  * @notice ERC20 Token backed by fiat reserves, version 2
  */
-contract FiatTokenV2 is FiatTokenV1_1, GasAbstraction, Permit {
-    bool internal _initializedV2;
```

Comment:
I've verified that this change is safe and backwards compatible.

Reply:
The uint8 will occupy the exact same number of bytes in the storage in the exact same position.